settings.gradle.kts

```diff
@@ -101,7 +101,7 @@ dependencyResolutionManagement {
             version("eddsa-version", "0.3.0")
             version("grpc-version", "1.50.2")
             version("guava-version", "31.1-jre")
-            version("hapi-version", "0.33.1-SNAPSHOT")
+            version("hapi-version", "0.34.0-SNAPSHOT")
```

Comment:
We should not depend on a snapshot version in the develop branch.

Reply:
Agreed, not sure that standard pattern for these dependencies, I was just following the existing pattern. We should fix this but not sure if it should block this PR.

Reply:
fixed